From bbbfebbbfd06cae898d5c218bab6e3f6c551edf7 Mon Sep 17 00:00:00 2001
From: Pavan S <psreenivas@habana.ai>
Date: Tue, 21 Feb 2023 08:10:08 +0200
Subject: [PATCH] [SW-123737] gaudi3: add empty reset file for eROM compilation

This patch adds empty reset file which will be used
for compilation of erom for adding reset support in
gaudi2 erom. The reset support for gaudi3 eROM will
be added later.

Signed-off-by: Pavan S <psreenivas@habana.ai>
Change-Id: Id62a089cfb04fe3fe0309c11d23c9563b50ea6ed
---

diff --git a/arch/gaudi3/common/reset.c b/arch/gaudi3/common/reset.c
new file mode 100644
index 0000000..ec9fc6f
--- /dev/null
+++ b/arch/gaudi3/common/reset.c
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: GPL-2.0+
+/*
+ * Copyright (C) 2023 HabanaLabs Ltd.
+ */
+/*
+ * TODO: Add gaudi3 specific eROM implementation
+ * Added empty source for gaudi3 eROM compilation .
+ */
